addons/google_calendar/models/calendar.py

```diff
@@ -15,6 +16,8 @@ class Meeting(models.Model):
 
     google_id = fields.Char(
         'Google Calendar Event Id', compute='_compute_google_id', store=True, readonly=False)
+    guests_can_modify = fields.Boolean(
+        'Guests Event Modification Permission', store=True, default=True, readonly=False)
```

Comment:
if it's not computed, store=True, readonly=False are not needed.
default=True will force to write True on almost all the events. it will fill the database column.
if the field was named differently, with a default value = False, we would not need to fill with data. Null will correspond to the default value. And it would not require an upgrade script. (you don't have one right now).